aws-alb-ingress-controller/Makefile

```diff
@@ -0,0 +1,14 @@
+.PHONY: apply
+apply:
+	@if ! kubectl get namespace "$$(basename $$PWD)$$(git rev-parse --short HEAD)" >/dev/null 2>&1; then \
+		kubectl create namespace "$$(basename $$PWD)$$(git rev-parse --short HEAD)"; \
+	fi
+	helm --kube-context $${KUBE_CONTEXT:-"dind"} --tiller-namespace $${TILLER_NAMESPACE:-"kube-system"} upgrade --wait --install --dry-run --namespace "$$(basename $$PWD)$$(git rev-parse --short HEAD)" $$(basename $$PWD)-$$(git rev-parse --short HEAD) --set readinessProbe.enable=false --set livenessProbe.enable=false .;
+
+.PHONY: test
+test:
+	echo "no test"
```

Comment:
@cw-sakamoto 
Can you write a test?
If you don't have a useful test, you don't need to write it.

Reply:
@cw-ozaki 

I don't think we can test this application because this application is work on AWS.
`/healthz` is confirmed port but including AWS authentication.

It looks like the test entry itself would fail [here](https://github.com/chatwork/charts/blob/master/.circleci/config.yml#L12) if it didn't exist, do you have it?